pkg/adapter/nimbus-kubearmor/processor/kspbuilder.go

```diff
@@ -264,6 +266,33 @@ func disallowChRoot() kubearmorv1.KubeArmorPolicy {
 	}
 }
 
+func disallowBinaries() kubearmorv1.KubeArmorPolicy {
+	return kubearmorv1.KubeArmorPolicy{
+		Spec: kubearmorv1.KubeArmorPolicySpec{
+			Process: kubearmorv1.ProcessType{
+				MatchPaths: []kubearmorv1.ProcessPathType{
+					{
+						Path: "/usr/sbin/*",
```

Comment:
This is going to prevent binaries from /usr/sbin/ from executing. I don't think that this is the behaviour that we want. We want to prevent binaries from temp or any other folder such as logs